src/test/scala/no/gdl/bookapi/service/translation/MergeServiceTest.scala

```diff
@@ -0,0 +1,61 @@
+/*
+ * Part of GDL book_api.
+ * Copyright (C) 2018 Global Digital Library
+ *
+ * See LICENSE
+ */
+
+package no.gdl.bookapi.service.translation
+
+import no.gdl.bookapi.{TestData, TestEnvironment, UnitSuite}
+import org.jsoup.Jsoup
+
+class MergeServiceTest extends UnitSuite with TestEnvironment {
+
+  private val service = new MergeService
+  private val originalHtml =
+    """<embed data-resource="image" data-resource_id="556"/>
+      |<embed data-resource="image" data-resource_id="557"/>
+      |<p>
+      | There was a large boulder on the grassy plain. He stumbled and fell down.
+      |</p>
+      |""".stripMargin
+
+  private val translatedHtml =
+    """<img src="https://images.test.digitallibrary.io/TYnPhXd4.jpg" />
+      |<img src="https://images.test.digitallibrary.io/abct.jpg" />
+      |<p>
+      | Hjorten traff en stor stein på den grønne sletten. Han snublet og falt.
+      |</p>
+      |""".stripMargin
+
+  test("that img-tags are replaced with embed-tags") {
+    val original = TestData.Domain.DefaultChapter.copy(content = originalHtml)
+    val translated = TestData.Crowdin.DefaultTranslatedChapter.copy(content = translatedHtml)
+
+    val newChapter = service.mergeChapter(original, translated)
+    val newContentDoc = Jsoup.parseBodyFragment(newChapter.content)
+    newContentDoc.select("embed[data-resource]").size() should be (2)
+    newContentDoc.getElementsByTag("img").size() should be (0)
+  }
```

Comment:
I would like to also assert that the order of images is kept intact. Suggested diff:
```
diff --git a/src/test/scala/no/gdl/bookapi/service/translation/MergeServiceTest.scala b/src/test/scala/no/gdl/bookapi/service/translation/MergeServiceTest.scala
index 760106c..6189781 100644
--- a/src/test/scala/no/gdl/bookapi/service/translation/MergeServiceTest.scala
+++ b/src/test/scala/no/gdl/bookapi/service/translation/MergeServiceTest.scala
@@ -19,6 +19,7 @@ class MergeServiceTest extends UnitSuite with TestEnvironment {
       |<p>
       | There was a large boulder on the grassy plain. He stumbled and fell down.
       |</p>
+      |<embed data-resource="image" data-resource_id="558"/>
       |""".stripMargin

   private val translatedHtml =
@@ -27,6 +28,7 @@ class MergeServiceTest extends UnitSuite with TestEnvironment {
       |<p>
       | Hjorten traff en stor stein på den grønne sletten. Han snublet og falt.
       |</p>
+      |<img src="https://images.test.digitallibrary.io/123.jpg" />
       |""".stripMargin

   test("that img-tags are replaced with embed-tags") {
@@ -35,7 +37,11 @@ class MergeServiceTest extends UnitSuite with TestEnvironment {

     val newChapter = service.mergeChapter(original, translated)
     val newContentDoc = Jsoup.parseBodyFragment(newChapter.content)
-    newContentDoc.select("embed[data-resource]").size() should be (2)
+    val embedElements = newContentDoc.select("embed[data-resource]")
+    embedElements.size() should be (3)
+    embedElements.get(0).toString should equal ("<embed data-resource=\"image\" data-resource_id=\"556\">")
+    embedElements.get(1).toString should equal ("<embed data-resource=\"image\" data-resource_id=\"557\">")
+    embedElements.get(2).toString should equal ("<embed data-resource=\"image\" data-resource_id=\"558\">")
     newContentDoc.getElementsByTag("img").size() should be (0)
   }

@@ -54,7 +60,7 @@ class MergeServiceTest extends UnitSuite with TestEnvironment {

     val newChapter = service.mergeChapter(original, translated)
     val newContentDoc = Jsoup.parseBodyFragment(newChapter.content)
-    newContentDoc.select("embed[data-resource]").size() should be (2)
+    newContentDoc.select("embed[data-resource]").size() should be (3)
     newContentDoc.getElementsByTag("img").size() should be (0)
   }
```